android/build.gradle

```diff
@@ -14,7 +14,7 @@ buildscript {
         // NOTE: Do not place your application dependencies here; they belong
         // in the individual module build.gradle files
         classpath 'org.jetbrains.kotlin:kotlin-gradle-plugin:1.4.32'
-        classpath 'org.greatfire:IEnvoyProxy:1.2.0'
+        classpath 'org.greatfire:IEnvoyProxy:1.3.1'
```

Comment:
is this upgraded only here?  doesn't seem related to this PR

Reply:
it's not really, but I endorse the change ;-) 1.3.1 has all the functionality we use, unaltered from 1.2.0, but has some bug fixes (I went to 1.3.0 when removing obfs4, we weren't using that anyway, but the change in feature set seemed like I should increase the version)

Reply:
if you want to be a stickler about it, we should make the change regardless :)